doc/reference/config/project_defaults.rst

```diff
@@ -0,0 +1,50 @@
+project_defaults
+----------------
+
+Specify default values for stanzas ``authors``, ``maintainers``, and ``license``
+of the dune-project file when initializing a project with ``dune init proj``.
```

Comment:
`dune-project` needs to be monospaced I think. Or link to the `dune-project` reference.